File: MainApp/app/src/beta/java/com/kolibree/android/app/dagger/VariantModule.java
package com.kolibree.android.app.dagger;

import com.kolibree.android.network.environment.DefaultEnvironment;
import com.kolibree.android.network.environment.Environment;
import dagger.Module;
import dagger.Provides;

@Module
public abstract class VariantModule {

  @Provides
  static DefaultEnvironment providesDefaultEnvironment() {
    return new DefaultEnvironment(Environment.STAGING);
  }
}

File: MainApp/app/src/release/java/com/kolibree/android/app/dagger/VariantModule.kt
/*
 * Copyright (c) 2020 Kolibree. All rights reserved
 *
 * Copying this file via any medium without the prior written consent of Kolibree is strictly
 * prohibited
 *
 * Proprietary and confidential
 */

package com.kolibree.android.app.dagger

import com.kolibree.android.network.environment.DefaultEnvironment
import com.kolibree.android.network.environment.Environment
import dagger.Module
import dagger.Provides

@Module
object VariantModule {
    @Provides
    fun providesDefaultEnvironment(): DefaultEnvironment {
        return DefaultEnvironment(Environment.PRODUCTION)
    }
}
